config/olm/manifests/bases/eventing-kogito.clusterserviceversion.yaml

```diff
@@ -75,4 +75,5 @@ spec:
   minKubeVersion: 1.20.0
   provider:
     name: Red Hat
+  replaces: eventing-kogito.v1.1.0
   version: 1.1.0
```

Comment:
This version should be raised too

Reply:
This is just the placeholder, the `operator-sdk` command replaces it for the right version: https://github.com/knative-sandbox/eventing-kogito/blob/main/Makefile#L188

Reply:
ok, good